### ISSUE TITLE
Screen reader should announce dropdown label , when focus on dropdown

### ISSUE THREAD

**Issue by dhakadgourav** [select2/select2#6134]
**Issue**: Screen reader users may not know about the dropdowns if the JAWS doesn't announce them to the user

**Responsible Service**: https://select2.org/appearance

Actual Result:​
JAWS is not announcing the dropdown options

**Expected Result**:​
JAWS should announce the dropdown options
 

**User Impact**:​
Screen reader users may not know about the dropdowns if the JAWS doesn't announce it to the user

**Comment by smalos** [comment]
This is already tracked here: https://github.com/select2/select2/issues/3744

**Comment by kevin-brown** [comment]
I'm going to keep this open since the one linked is a multi-ticket (covers a wide variety of issues through the years) and this one is a bit more focused on the specific issue with labels.
